multiple_buyers/entry.json

```diff
@@ -0,0 +1,14 @@
+[{
+  "name": {
+    "en" : "Multiple_Buyers_Extension"
```

Comment:
The underscore is not needed in the name here. 

Suggested name for this extension is 'Multiple buyers per contract'